test/runner.sh

```diff
@@ -28,7 +28,10 @@ mkdir -p dump
 # Hack to grant TEST_PGUSER superuser status so that we can
 # consistently run tests using the same user rather than the
 # current/local user
-${PSQL} $@ -v ECHO=none -c "ALTER USER ${TEST_PGUSER} WITH SUPERUSER;"
+${PSQL} $@ -U `whoami` -v ECHO=none -c "ALTER USER super_user WITH SUPERUSER;"
```

Comment:
No need to do `whoami`, the host user is available as `$USER` in shells.  I'd prefer we pass on the `super_user` as an exported shell variable (`$PG_SUPERUSER`) by exporting it in the Makefile (providing a default value in this script) and then further exporting to `psql` as an env variable with `-v SUPERUSER=super_user`.